atomic_reactor/plugins/pre_fetch_sources.py

```diff
@@ -222,6 +222,32 @@ def get_remote_urls(self):
 
         return remote_sources_urls
 
+    def get_blacklisted_sources(self):
+        src_config = get_source_container(self.workflow, fallback={})
+        blacklist_srpms = src_config.get('blacklist_srpms')
+        if not blacklist_srpms:
+            self.log.debug('blacklist_srpms is not defined in reactor_config_map')
+            return []
+
+        blacklist_url = blacklist_srpms.get('blacklist_url')
+        blacklist_key = blacklist_srpms.get('blacklist_key')
```

Comment:
Is there a reason to use `get()` here? They are required in the schema anyway, right?

Reply:
right will remove